modules/markdown.php

```diff
@@ -6,7 +6,7 @@
  * Sort Order: 31
  * First Introduced: 2.8
  * Requires Connection: No
- * Auto Activate: No
+ * Auto Activate: Yes
```

Comment:
It won't change the content unless user saves post again (and post has Markdown content of course). See `WPCom_Markdown->load_markdown_for_posts()`